projects/fsastorefront-e2e-cypress/cypress/helpers/fnol.ts

```diff
@@ -233,12 +221,119 @@ export function checkDownloadButton() {
+      cy.get('div.info-card-links').contains('Add Documents');
+    });
+}
+
+export function uploadDocuments() {
+  const filePath = 'Claim.pdf';
+  cy.get('.custom-file-input').attachFile(filePath);
+  cy.get('.btn-primary')
+    .should('contain.text', 'Upload')
+    .eq(0)
+    .click({ force: true });
+}
+
+export function startClaimFromHomepage() {
+  cy.get('cx-banner').eq(2).click();
+  cy.get('.section-header-heading').contains('Which car has been damaged?');
+  cy.contains('The affected car is not listed? Contact an agent for help!');
+  cy.get('.form-check-label').should('be.visible');
+  cy.get('.primary-button').contains('Continue').should('be.disabled');
+  cy.get('.heading-headline')
+    .should('be.visible')
+    .contains('Make a Claim Online');
+}
+
+export function selectAutoPolicyAndStartClaim() {
+  cy.get('cx-card')
+    .should('be.visible')
+    .within(() => {
+      cy.contains('Auto Insurance Policy');
+      cy.contains('Vehicle Make: Tesla');
+      cy.get('.btn-link').contains('Select').click({ force: true });
+    });
+  cy.get('.form-check-input').click();
+  cy.get('.primary-button').contains('Continue').click();
+}
+
+export function deleteClaim() {
+  cy.get('.info-card').within(() => {
+    cy.contains('Fire')
+      .parents('.col-12')
+      .within(() => {
+        cy.get('a.fs-icon.icon-bin')
+          .should('be.visible')
+          .eq(0)
+          .click({ force: true });
+      });
+  });
+  cy.get('cx-fs-deleted-claim-dialog', { withinSubject: null })
+    .should('be.visible')
+    .within(() => {
+      cy.get('h3').contains('Delete started claim process');
+      cy.get('.action-button').contains('Cancel');
+      cy.get('button.close').should('be.visible');
+      cy.get('.primary-button').contains('Delete').click();
+    });
+}
+
+export function claimAction(action) {
+  cy.get('.info-card').within(() => {
+    cy.contains('Glass Damage')
+      .parents('.col-12')
+      .within(() => {
+        cy.get('a.link').contains(action).click({ force: true });
+      });
+  });
+}
+
+export function checkUpdateClaimPage() {
+  cy.get('cx-form-component', { withinSubject: null }).should('be.visible');
+  cy.get('cx-fs-change-claim-navigation', { withinSubject: null })
+    .should('be.visible')
+    .within(() => {
+      cy.get('.action-button').contains('Back');
+      cy.get('.primary-button').contains('Submit');
+    });
+  cy.get('.heading-headline').contains('Update Claim');
+}
+
+export function uploadNewDocument(newDocument) {
+  cy.get('cx-form-component', { withinSubject: null })
+    .should('be.visible')
+    .within(() => {
+      const filePath = newDocument;
+      cy.get('.col-form-label').contains(
+        'Please upload files relevant to the incident:'
+      );
+      cy.get('.custom-file-input').attachFile(filePath);
+      cy.get('.btn-primary')
+        .should('contain.text', 'Upload')
+        .click({ force: true });
+      cy.get('.fas.fa-check').should('be.visible');
```

Comment:
`.fa-check` should be enough.

Reply:
right, tried and it is working - fas removed